### ISSUE TITLE
Improve reconnection logging

### ISSUE THREAD

**Issue by antobinary** [bigbluebutton/bigbluebutton#14456]
**Describe the bug**
I got reconnected in a 2.4.4 server meeting.
First few seconds - webcams froze, sound stopped.

Next: webcams turned black with spinning wheel.
Next: the presentation area went missing
![Screenshot from 2022-02-24 13-35-23](https://user-images.githubusercontent.com/6312397/155586183-865c1e9e-a3ad-4899-a503-2a5514c2b42d.png)

Next: my client looked like it reloaded. I could see webcams streams again, people could hear me, I could not hear them. Chat, etc worked.

My connection status after the reload was green, and the list of the history was empty.

My browser logs showed sip websocket disconnection. (?)
I searched through the server logs using my name -> found the userid.

When I parsed for `Connection status updated` I only saw one entry for my userid, the `recovered` one...
Later encountered 

```
Feb 24 16:57:26 redacted.com systemd_start_frontend.sh[8676]: info: Removing connectionId for user. sessionId=a250271b966cdec10711ba127c20b7a0ebe277ba-1645720152868--w_pyob3kblobuy connectionId=pRB7Mnb8s4DdPXAc8
```

```
Feb 24 16:57:34 redacted.com systemd_start_frontend.sh[8676]: info: Upserted {"meetingId":"a250271b966cdec10711ba127c20b7a0ebe277ba-1645720152868","userId":"w_pyob3kblobuy","connectionId":"gcFHBpB2wGuDZyE7Z"} 2 in AuthTokenValidation
Feb 24 16:57:34 redacted.com systemd_start_frontend.sh[8676]: info: User 'w_pyob3kblobuy' is trying to validate auth token for meeting 'a250271b966cdec10711ba127c20b7a0ebe277ba-1645720152868' from connection 'gcFHBpB2wGuDZyE7Z'
Feb 24 16:57:34 redacted.com systemd_start_frontend.sh[8676]: info: Client connections add called {"logCode":"client_connections_add","extraInfo":{"sessionId":"a250271b966cdec10711ba127c20b7a0ebe277ba-1645720152868--w_pyob3kblobuy","connection":{"id":"gcFHBpB2wGuDZyE7Z","clientAddress":"127.0.0.1","httpHeaders":{"host":"poolhtml5servers","user-agent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.80 Safari/537.36","accept-language":"en,en-GB;q=0.9,en-US;q=0.8"}}}}
Feb 24 16:57:34 redacted.com systemd_start_frontend.sh[8676]: info: Adding new connection for sessionId=a250271b966cdec10711ba127c20b7a0ebe277ba-1645720152868--w_pyob3kblobuy connection=gcFHBpB2wGuDZyE7Z
```

### Questions (so we can see what we can improve)

1) Why logs did not show `Connection status updated` for me losing connection?
We could probably add log where we do `info: Removing connectionId for user. ` and also where we do `info: Adding new connection for` so that when we grep for `Connection status updated` we see
when connection was established (with the userid and connectionid)
when connection deteriorated
when connection recovered (if?)
when connection was closed
and possibly do these steps for each connection the user held...

2) Why was the user connection status icon green and why was the list empty as if all was dandy? Do we perhaps list based on the old connectionId or based on the new one which was all good? Or we group per userid?

3) Looked like multiple sockets got closed or disrupted. We could possibly display more info in the browser?

4) Why did the presentation area disappear?

5) Audio state seemed to show me as joined with mic, unmuted during the reconnection (8s+) and upon reconnection I had to mute myself. I could not hear. I am pretty sure I was in with mic but muted state prior to the disconnection. And why could I not hear after?

**Comment by danimo** [comment]
We observed the same behavior in our cluster.

**Comment by antonbsa** [comment]
I got the same behavior (**v2.5**):
- First ~5-10 seconds: entire client just froze.
- After that, my client reloaded.
- Then it looked like my audio was connected but it wasn't (michrophone button just disappears). I tried to speak but no one heard me. I also tried clicking the audio button to reconnect but this has no effect.
- I had to refresh the page.

![image](https://user-images.githubusercontent.com/56703993/155765400-ade67d38-e2de-4bc5-a8a1-87393bb91440.png)
